index.html

```diff
@@ -50,6 +50,185 @@ <h1>Hey there. I'm Nicolas<br><span>I'm a software developer</span></h1>
+                            <li><p class="works-languajes">html</p></li>
+                        </ul>
+                        <a href="#" class="button">See Project</a>
+
+                    </div>
+
+                </div>
+
+                <div class="work">
+                    <div class="img-placeholder"></div>
+                    
+                    <div class="work-info">
+                        <h4>Multi-Post Stories Gain+Glory</h4>
+                        <ul class="works-languajes-container">
+                            <li><p class="works-languajes">Ruby on rails</p></li>
+                            <li><p class="works-languajes">css</p></li>
+                            <li><p class="works-languajes">JavaScript</p></li>
+                            <li><p class="works-languajes">html</p></li>
+                        </ul>
+                        <a href="#" class="button">See Project</a>
+
+                    </div>
+
+                </div>
+
+                <div class="work">
+                    <div class="img-placeholder"></div>
+                    
+                    <div class="work-info">
+                        <h4>Multi-Post Stories Gain+Glory</h4>
+                        <ul class="works-languajes-container">
+                            <li><p class="works-languajes">Ruby on rails</p></li>
+                            <li><p class="works-languajes">css</p></li>
+                            <li><p class="works-languajes">JavaScript</p></li>
+                            <li><p class="works-languajes">html</p></li>
+                        </ul>
+                        <a href="#" class="button">See Project</a>
+
+                    </div>
+
+                </div>
+
+                <div class="work">
+                    <div class="img-placeholder"></div>
+                    
+                    <div class="work-info">
+                        <h4>Multi-Post Stories Gain+Glory</h4>
+                        <ul class="works-languajes-container">
+                            <li class="works-languajes">Ruby on rails</li>
+                            <li class="works-languajes">css</li>
+                            <li class="works-languajes">JavaScript</li>
+                            <li class="works-languajes">html</li>
+                        </ul>
+                        <a href="#" class="button">See Project</a>
+
+                    </div>
+
+                </div>    
+
+            </div>
+        </section>
+
+        <section id="aboutme-section">
+            <h2>About me</h2>           
+            <img src="images/Mask Group.png" alt="vector" class="vector">
+            <p> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don't hestiate to contact me.</p>
+            <a href="#" class="button">Get My Resume</a>
+
+            <div class="aboutme-card">
```

Comment:
###### Great job adding the decoration, But I need to mention two points here. 
- [ ] One: your first decoration image is not a background image, remember decorations are always backgrounded images, not `img` tags. 
- [ ] Two: your second image is expected to stick to the `About Me` content, as you can see from the screenshot I added it moves down to the other contents. Please address this for a better layout. 

![Screenshot 2023-01-13 202937](https://user-images.githubusercontent.com/69077061/212382736-01c42f95-7e62-4cdd-ad8e-b3fed222ef20.png)